cookbook.go

```diff
@@ -73,8 +82,9 @@ type Cookbook struct {
 	Libraries    []CookbookItem `json:"libraries,omitempty"`
 	Providers    []CookbookItem `json:"providers,omitempty"`
 	Resources    []CookbookItem `json:"resources,omitempty"`
-	RootFiles    []CookbookItem `json:"templates,omitempty"`
+	RootFiles    []CookbookItem `json:"root_files,omitempty"`
 	Metadata     CookbookMeta   `json:"metadata,omitempty"`
+	Access       CookbookAccess `json:"access,omitempty"`
```

Comment:
@afiune, @MarkGibbons  could you please explain what does this `Access` field means https://docs.chef.io/server/api_chef_server/#get-23?

Reply:
@svagner Could you open an issue for this question?  I'll try to look over the weekend, but a comment on a closed PR is likely to get lost.  thanks

Reply:
Sure thing!